feature-libs/product/public_api.ts

```diff
@@ -1,5 +1,4 @@
 /*
  * Public API Surface of product
  */
-
-export * from '@spartacus/product/configurators';
+export * from './dummy-product.module';
```

Comment:
I hope we can drop now this module from public api - so when it's released with actual useful modules, no one forgets to remove it ;)

Reply:
But we need to have one line in this file..otherwise we need to drop the file and its references. So I would propose to keep it as is

Reply:
Just export a dummy constant, add a comment